### PR TITLE
Add fixtures recipe

### LINKED ISSUE #244: Show example with loading multiple fixtures
- loading fixtures before each test
- loading fixtures once and saving in closure variables
- loading fixtures once and saving in the `this` context

### PR DESCRIPTION
**Pull request by bahmutov**
- closes #244

### CODE REVIEW

File: examples/fundamentals__fixtures/cypress/integration/load-fixtures-spec.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        country = c
+      })
+    })
+
+    it('has loaded fixtures', () => {
+      expect({ city, country }).to.deep.equal({
+        city: { name: 'Atlanta' },
+        country: { name: 'United States' }
+      })
+    })
```
Comment (jennifer-shehane) [inline examples/fundamentals__fixtures/cypress/integration/load-fixtures-spec.js]: are you positive this works when you add another test?

File: examples/fundamentals__fixtures/cypress/integration/multiple-fixtures-spec.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        // thanks to JavaScript closures we have access to both fixtures
+        expect({ city, country }).to.deep.equal({
+          city: { name: 'Atlanta' },
+          country: { name: 'United States' }
+        })
+      })
+    })
+  })
+
+  it.skip('loads fixtures using Cypress.Promise.all - not recommended due to race condition', () => {
```
Comment (jennifer-shehane) [inline examples/fundamentals__fixtures/cypress/integration/multiple-fixtures-spec.js]: I think this feels a bit weird - to add an example of an antipattern into the example recipes.

Comment (bahmutov) [reply examples/fundamentals__fixtures/cypress/integration/multiple-fixtures-spec.js]: sure I can remove it - but people will think `why can't I use Promise.all` ...

File: examples/fundamentals__fixtures/cypress/integration/load-fixtures-spec.js (hunk 1 above)
Comment (bahmutov) [reply examples/fundamentals__fixtures/cypress/integration/load-fixtures-spec.js]: yup, and I have added second tests to confirm this

File: examples/fundamentals__fixtures/cypress/integration/multiple-fixtures-spec.js (hunk 2 above)
Comment (lorenzogrv) [reply examples/fundamentals__fixtures/cypress/integration/multiple-fixtures-spec.js:16]: > sure I can remove it - but people will think `why can't I use Promise.all` ...

That's exactly how I reached this notes

**Pull request merged**